geoips/plugins/yaml/workflows/read_test_v2alpha1.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: geoips/v2alpha1
+interface: workflows
+test: v2alpha1
+family: order_based
+name: read_test_v2alpha1
+docstring: |
+  11.2 µm Infrared.
+
+  This product utilizes channel 14 (11.2 µm) and highlights areas of deep convection within a Tropical Cyclone.
+description: 11.2 µm Infrared.
+relpath: geoips/plugins/yaml/workflows/read_test.yaml
+abspath: home/kumar/geoips/geoips/geoips/plugins/yaml/workflows/read_test.yaml
```

Comment:
Should the abs path be included in the yaml? Or should it be auto generated with the plugin registry?